src/standard/types/Promisable.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Drash - A microframework for building JavaScript/TypeScript HTTP systems.
+ * Copyright (C) 2023  Drash authors. The Drash authors are listed in the
+ * AUTHORS file at <https://github.com/drashland/drash/AUTHORS>. This notice
+ * applies to Drash version 3.X.X and any later version.
+ *
+ * This file is part of Drash. See <https://github.com/drashland/drash>.
+ *
+ * Drash is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Drash is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * Drash. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+export type Promisable<T> = Promise<T> | T;
```

Comment:
I see. How do you unwrap the typeof function to handle it synchronous or asynchronous? In my original implementation I just said: "Whatever, I'll just make them use Promises all the way".

Reply:
i agree with "use promises all the way" and originally this was a type that handled resources like in v2 where you could opt to use async or not. so it stayed and was left intact even though promises are being used by default. that being said, this type doesn't make sense. we good with removing it?

Reply:
If we are not handling sync vs async differently, like with a weird hack of `typeof func === "[AsyncFunction]"`, and we are just awaiting always, I think we should remove it.
Honestly, in my opinion, the only instance where synchronous context makes sense is on application startup, otherwise async is required.

Reply:
removed in [dc78e87](https://github.com/drashland/drash/pull/698/commits/dc78e87eae57b62cff7b1ec693f5bca3c2ce31da)